components/gitpod-cli/BUILD.yaml

```diff
@@ -8,5 +8,9 @@ packages:
     env:
       - CGO_ENABLED=0
       - GOOS=linux
+    deps:
+      - components/common-go:lib
```

Comment:
Not sure the `common-go` dependency is needed. No API package should have a common-go dependency, and `gitpod-cli` does not need it (see its go.mod).

Reply:
I removed it here: https://github.com/gitpod-io/gitpod/pull/3569/commits/fa9cace2b1cdfdb8e0f27d54b896490c91fa71f7

Reply:
OK

Reply:
Let's fix that with your PR then.